build/ansible/roles/postgres/tasks/main.yml

```diff
@@ -18,6 +18,8 @@
       - percona-postgresql14
       - python3-psycopg2 # Python PostgreSQL database adapter
     state: installed
+    nobest: true
```

Comment:
Yum is redirected to dnf, so that should be no problem. When it comes to installing the non-latest version - thanks for pointing that out. A fully agree, it's not a proper fix.

I believe we should leave it as is, no modification is needed.

Reply:
> I believe we should leave it as is, no modification is needed

Before PR or changes in PR? Does it worth to risk by having non latest version of PG to remove telemetry agent?

Reply:
No, I mean we should close this PR so the code is unchanged.